app/src/components/ArticlesPublications.jsx

```diff
@@ -23,26 +51,40 @@ class ArticlesPublications extends Component {
     let pageContent = [];
 
     if (articlesPublications && articlesPublications.articles) {
-      const articles = articlesPublications.articles.map(this.renderElement);
+      const publications = articlesPublications.publications.map(this.renderElement);
 
-      if (articles && articles.length) {
```

Comment:
no, in here we had articles and you replaced with publications. In line 76, we had publications and you replaced with articles. code is the same, you just changed the order for some reason